wordcloud/wordcloud.py

```diff
@@ -538,7 +540,13 @@ def generate_from_text(self, text):
         -------
         self
         """
-        words = self.process_text(text)
```

Comment:
I know. But I didn't want to expose it in wordcloud_cli.py. Do you think its okay?